src/Pyz/Zed/Console/ConsoleDependencyProvider.php

```diff
@@ -400,6 +400,8 @@ protected function getConsoleCommands(Container $container): array
             $commands[] = new GenerateRestApiDocumentationConsole();
             $commands[] = new QueueDumpConsole();
             $commands[] = new EventTriggerListenerConsole();
+            $commands[] = new CustomerPasswordResetConsole();
+            $commands[] = new CustomerPasswordSetConsole();
```

Comment:
Why are those added for Development mode only?
Shouldnt they be present also in prod?

Reply:
Damn, nice catch! I'll request for a fix.